packages/wonder-blocks-dropdown/src/components/dropdown-opener.tsx

```diff
@@ -56,18 +64,19 @@ class DropdownOpener extends React.Component<Props> {
     ): React.ReactElement {
         const {disabled, testId, text} = this.props;
         const renderedChildren = this.props.children({...eventState, text});
-        const childrenProps = renderedChildren.props;
-        const childrenTestId = this.getTestIdFromProps(childrenProps);
+        const rendererdChildrenProps: RenderedChildrenProps =
+            renderedChildren.props; // type-coverage:ignore-line
```

Comment:
I wasn't able to cover this line with proper types.  Thankfully the type coverage tooling supports coverage exclusion comments.  The reported generated by `yarn coverage:types` uses the same underlying tooling so these comments should be respected when that report is generated.